test/e2e/broker_event_transformation_test.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 /*
-TestEventTransformationForTrigger tests the following scenario:
```

Comment:
And from now one we have test renaming... Again, unrelated to this PR and I think they deserve their own PR (and proper explanation).

Oh, and particularly for this one, once again I wonder why you're switching to using a more obscure abbreviation instead of the full words in the test name.